test/unit/math/prim/fun/inv_Phi_test.cpp

```diff
@@ -19,6 +19,39 @@ TEST(MathFunctions, inv_Phi) {
   p = 0.97575;
   EXPECT_FLOAT_EQ(p, Phi(inv_Phi(p)));
 }
+
+TEST(MathFunctions, Equal) {
+  using stan::math::inv_Phi;
+  // test output generated with WolframAlpha
+  double p[]
+      = {0.0000001, 0.00001, 0.001, 0.05, 0.15, 0.25,  0.35,    0.45,
+         0.55,      0.65,    0.75,  0.85, 0.95, 0.999, 0.99999, 0.9999999};
+
+  double exact[]
+      = {-5.199337582192816931587347266962336866509737160238716454318950531,
```

Comment:
This won't hurt anything, but they can also be cut down to 18 digits of precision as that's already more than you get with `double`.